astropy/io/fits/tests/test_image.py

```diff
@@ -925,9 +925,6 @@ def test_image_update_header(self):
         with fits.open(self.temp("test0.fits")) as hdul:
             assert (orig_data == hdul[1].data).all()
 
-    # The test below raised a `ResourceWarning: unclosed transport` exception
-    # due to a bug in Python <=3.10 (cf. cpython#90476)
-    @pytest.mark.filterwarnings("ignore:unclosed transport <asyncio.sslproto")
```

Comment:
Since this cleanup is actually orthogonal, I've opened https://github.com/astropy/astropy/pull/16183 for it.